shared/uavobjectdefinition/stabilizationsettings.xml

```diff
@@ -27,6 +27,9 @@
 		<field name="PitchPI" units="" type="float" elementnames="Kp,Ki,ILimit" defaultvalue="2.5,0,50" limits="%BE:0:20,%BE:0:20,"/>
 		<field name="YawPI" units="" type="float" elementnames="Kp,Ki,ILimit" defaultvalue="2.5,0,50" limits="%BE:0:20,%BE:0:20,"/>
 		<field name="GyroCutoff" units="Hz" type="float" elements="1" defaultvalue="55.0"/>
+		<field name="GyroButterworth" units="" type="enum" elements="1" options="FALSE,TRUE" defaultvalue="FALSE">
```

Comment:
Any chance we could go a bit further and let the user select a filter order here, and dynamically construct the right thing?  For helis, second order will be a big improvement but I think they might really want 4th-5th order..